src/containers/app/CreateAppType.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FormControl, FormGroup, ControlLabel, Button, Alert } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import AsyncContent, { AsyncStatePropType } from '../../components/asynccontent/AsyncContent';
+import { createAppTypeRequest } from './AppActionFactory';
+
+
+class CreateAppType extends React.Component {
+
+  static propTypes = {
+    actions: PropTypes.shape({
+      onCreateAppType: PropTypes.func.isRequired
+    }).isRequired,
+    createAppTypeAsyncState: AsyncStatePropType.isRequired
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      description: '',
+      name: '',
+      namespace: '',
+      entityTypeId: null
+    };
+  }
+
+  componentDidMount() {
+  }
+
+  onTitleChange = (event) => {
+    this.setState({
+      title: event.target.value
+    });
+  };
+
+  onNameChange = (event) => {
+    this.setState({
+      name: event.target.value
+    });
+  };
+
+  onNamespaceChange = (event) => {
+    this.setState({
+      namespace: event.target.value
+    });
+  };
+
+  onDescriptionChange = (event) => {
+    this.setState({
+      description: event.target.value
+    });
+  };
+
+  onEntityTypeIdChange = (event) => {
+    this.setState({
+      entityTypeId: event.target.value
+    });
+  }
+
+  onSubmit = () => {
+    const { title, description, name, namespace, entityTypeId } = this.state;
+    const AppType = {
```

Comment:
This is fine for now, but we should add `AppType` and `AppTypeBuilder` in `lattice-js`.